### ISSUE TITLE
HTML5 player not resizing to "fill" setting

### ISSUE THREAD

**Issue by Rauvagol** [YePpHa/YouTubeCenter#1338]
Still appears as 720 p box, regardless of "fullscreen top player" setting or not

http://i.imgur.com/JzmOelX.jpg

**Comment by Eisys** [comment]
What version are you using? This is a bug with older version! You should use the latest DEV version!
https://github.com/YePpHa/YouTubeCenter/wiki/Developer-Version

**Comment by Rauvagol** [comment]
Ah, I didnt realize the main version was not up to date with the changelog, thanks for the info.

**Comment by Yonezpt** [comment]
Try the latest stable version first, it's 2.1.7 and can be downloaded here: https://github.com/YePpHa/YouTubeCenter/wiki

Do not use the AMO link since that leads to an outdated version.

If the problem still persists with that version then use the one suggested by @Eisys

**Comment by Rauvagol** [comment]
Yeah, I have 2.1.7 and that bug still exists for me, so dev it is.

**Comment by Yonezpt** [comment]
Let us know if the problem is fixed for you with the developer version.

**Comment by Rauvagol** [comment]
Looks fixed to me, thanks

**Comment by Vashetti** [comment]
Still occurs for me on the latest Dev version with Safari.

**Comment by Yonezpt** [comment]
@Vashetti Are you using it with SPF enabled or disabled? If you have it disabled could you try with it enabled?

**Comment by Vashetti** [comment]
Had it disabled, just enabled it, first video I went on, the problem occurred. 

Despite being on the latest Dev version, is it entirely possible I also have a previous version of YTC installed? In my extension list on Safari it only lists the 461 Dev version, but I was thinking there could be a conflict or something. Because I see many people nowadays saying to update to the latest dev version and that the bug was squashed a while ago, yet it's been occurring for me for ages.

**Comment by Yonezpt** [comment]
That's normal because YTC saves your settings in a local file, so if you remove the addon it still retains your previous settings. Whenever you are in doubt regarding the YTC version you can simply open the settings menu and read the current version on the bottom left corner, that way you are sure of which version you are using.

Regarding the actual issue, use the latest Dev version instead of the v2.1.7, it has been confirmed that this issue is present in 2.1.7.

After you reinstall the latest Dev version, could you try resetting your settings to their default, switch the player to html5 (if it isn't already by default) and copy paste the console log into a new gist? https://gist.github.com/

If you don't know how to open the console: http://webmasters.stackexchange.com/questions/8525/how-to-open-the-javascript-console-in-different-browsers

**Comment by Vashetti** [comment]
Here's one with default settings (HTML5, Player size set to 80%): https://gist.github.com/Vashetti/1b3aa71a35665f70ef9b

**Comment by Yonezpt** [comment]
Yep, something is not working correctly, the fix is not being applied on your end. If it was then your log should have these lines included:

````javascript
"ytplayer.load() has been called." watch:1980
"yt.player.Application.create has been called" watch:10742
"Pointer: j" watch:1980
"Progressbar: xb" watch:1980
"Canvas: ab"
````

I don't know what else you can do except use the flash player until YePpHa can take a look at what is causing this.

Another thing you can try (although I suspect it won't make a difference in this case) is to disable any other addon that you have running except YTC and try again. With some unlikely luck it is some addon that is causing a conflict in this particular case.

**Comment by Vashetti** [comment]
I hope it's an easy issue to fix.

**Comment by Yonezpt** [comment]
The fix already exists, I suspect is causing it to fail when YTC tries to apply it. If you try the latest dev version with Firefox or Chrome you will see that the HTML5 player scales correctly.

**Comment by Vashetti** [comment]
Ah right. I hope it's a simple process to get the fix applied within the Safari version then :)

**Comment by 8BitCarnage** [comment]
Having the same issue with Chrome and the HTML5 player running the userscript version of 2.1.7. Tried disabling all add-ons, that did nothing. Have reset settings multiple times to no avail.

Tried installing the #471 Dev build, but I got a different bug where the player would sit in front of everything else and would stay static (not sure how to describe it) on the screen when scrolling. Tried to reset the settings to default but the page would lock up after clicking "Save & Reload/Reset", and upon manually refreshing nothing was applied. Tried disabling all extension once again, nothing changes. 

Here's a shot of the Dev build problem, I tried to demonstrate it as best as I could:

![image](https://cloud.githubusercontent.com/assets/10793741/5991284/4f80c8c2-aa37-11e4-90db-359322816a02.png)

I'll just go back to using the Flash version I guess, just wanted to report my findings if they could be of any help.

**Comment by YePpHa** [comment]
@8BitCarnage This is caused by the option `Enable player docking`, which should have been disabled by default. Try disabling that option and see if that fixes the player's position.

For the reset settings not working has been fixed in the newest developer version.

**Comment by 8BitCarnage** [comment]
@YePpHa Yep, build #477 is now working perfectly. Had to reset settings because the player size wouldn't go past ~50% fill no matter what option was chosen, whether it were in percent or pixels, but now it's working as it did a few months ago. Thanks heaps.

**Comment by Vashetti** [comment]
Just got build #486 for Safari, and I don't want to jump the gun here, but it seems as though my issue is fixed, videos are filling the player properly.

**Comment by andrewjmetzger** [comment]
@YePpHa Confirmed this is a persisting issue in chrome 43.0.2357.81, version dev. #525 with docking disabled. A workaround *\*grumble\** is to "Disable player resizing" in the resize tab, but if you could, a fix would be awesome, thanks!  

PS: I am happy to help debug if needed.

**Comment by 8BitCarnage** [comment]
Yep, this bug is back as of 525 :(
